### ISSUE TITLE
support Oracle named parameters enclosed in double quotes

### ISSUE THREAD

**Issue by dedabob** [cpburnz/python-sqlparams#11]
According to standard SQL identifiers enclosed in double quotes are case sensitive while those not enclosed must be converted to uppercase. Oracle applies that rule to parameters too.

These code snippets work fine with both cx-oracle and oracledb:
```python
cursor.execute('select * from "users" where "id" = :id', {'ID': 1})
```
```python
cursor.execute('select * from "users" where "id" = :ID', {'id': 1})
```
```python
cursor.execute('select * from "users" where "id" = :"ID"', {'id': 1})
```
```python
cursor.execute('select * from "users" where "id" = :"id"', {'"id"': 1})
```
These snippets, instead, would raise a _DPY-4008: no bind placeholder_ DatabaseError:
```python
cursor.execute('select * from "users" where "id" = :"id"', {'id': 1})
```
```python
cursor.execute('select * from "users" where "id" = :id', {'"id"': 1})
```

NamedToNamedConverter should take that into account, maybe adding an "named_oracle" style.

**Comment by cpburnz** [comment]
@dedabob Thanks for proposing this improvement. I'll review your pull request when I have time.

**Comment by cpburnz** [comment]
This feature has been released in v6.1.0 which has been published to PyPI.
